packages/dd-trace/src/config.js

```diff
@@ -175,6 +191,37 @@ class Config {
       process.env.DD_TRACE_EXPERIMENTAL_TRACEPARENT_ENABLED,
       false
     )
+    // TODO: change this to traceparent,datadog
+    // Traceparent should be prioritized over datadog headers, but this is a breaking change until
+    // we support tracestate headers too, otherwise we would lose origin and tags header data.
+    const defaultPropagationStyle = ['datadog']
+    if (isTrue(DD_TRACE_TRACEPARENT_ENABLED)) {
```

Comment:
According to the API docs above, the default includes this (although it says `traceparent` in the API docs). Should this be enabled by default, or should the API doc be changed? (might be both too since the name is different between them).

Reply:
The plan was to leave it behind the experimental flag until the `tracestate` part is complete as I was aiming to try and land this separately from the `tracestate` support, but they can be landed together and still not be a breaking change so long as we retain the current ordering until the next major. I'm thinking it will be better to just add the `tracestate` part to it now and unflag it as part of that. The main reason I was trying to split it up before was because we seemed to want to reprioritize on OTel stuff, but now we're back to this being the priority so I might at well just do the rest here.